src/main/java/com/eleks/academy/whoami/core/impl/TurnImpl.java

```diff
@@ -61,6 +85,18 @@ public void action(String player, String value) {
 		});
 	}
 
+	@Override
+	public boolean calculateAnswers() {
+		long yes = this.getCurrentTurn().stream()
```

Comment:
you can filter by Yes and Not sure in one stream, no need to count them separately if in the end you sum them together

Reply:
Yes firstly I try to do this, but I use operator && and it didn't work. Now I understand that it always was false. Now I change it to ||